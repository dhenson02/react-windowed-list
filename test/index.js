// test
import test from 'ava';

// src
import WindowedList from 'src/index';

test.todo('WindowedList');
