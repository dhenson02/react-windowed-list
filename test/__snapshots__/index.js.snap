// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`if WindowedList renders correctly when type is not simple 1`] = `
<div
  style={
    Object {
      "height": 500,
    }
  }
>
  <Measured(WindowedList)
    length={1000}
    type="uniform"
  >
    <WindowedList
      axis="y"
      itemRenderer={[Function]}
      itemsRenderer={[Function]}
      length={1000}
      pageSize={10}
      size={
        Object {
          "height": 0,
          "width": 0,
        }
      }
      threshold={100}
      type="uniform"
      usePosition={false}
      useStaticSize={false}
      useTranslate3d={false}
    >
      <div
        style={
          Object {
            "height": "100%",
            "width": "100%",
          }
        }
      >
        <div
          style={
            Object {
              "position": "relative",
            }
          }
        >
          <div
            style={
              Object {
                "WebkitTransform": "translate(0px, 0px)",
                "msTransform": "translate(0px, 0px)",
                "transform": "translate(0px, 0px)",
              }
            }
          >
            <div>
              <div>
                0
              </div>
            </div>
          </div>
        </div>
      </div>
    </WindowedList>
  </Measured(WindowedList)>
</div>
`;

exports[`if WindowedList renders correctly with default props 1`] = `
<div
  style={
    Object {
      "height": 500,
    }
  }
>
  <Measured(WindowedList)
    length={1000}
  />
</div>
`;
